Runtime.sln

```diff
@@ -1,8 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30501.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.27016.1
+MinimumVisualStudioVersion = 15.0
```

Comment:
FYI I edited the .sln files then made temporary updates in VS to see if it wanted anything else e.g. the `$(SolutionGuid)` additions.